src/System.IO.IsolatedStorage/src/project.json

```diff
@@ -8,13 +8,16 @@
         "System.Diagnostics.Tools": "4.3.0-beta-24604-02",
         "System.IO": "4.3.0-beta-24604-02",
         "System.IO.FileSystem": "4.3.0-beta-24604-02",
+        "System.IO.FileSystem.AccessControl": "4.3.0-beta-24604-02",
```

Comment:
We would have to split the library between Windows and Linux in order to do independent dependencies, but that is a potential option.

>related to invoking them would be significantly/meaningfully smaller than the AccessControl library?

Yes my hope was that it would be simpler and smaller but it doesn't sound like it is. We can probably live with this dependency but it will be on my watch list when I look at the closure of all the libraries.